### PR TITLE
Fix proguard. Added three donotwarn for things we don't use at all in…

### PR DESCRIPTION
**Pull request by zsurocking**
… this testing app.

### CODE REVIEW

**Comment by zsurocking** [comment]
#1614

**Comment by carl-mastrangelo** [comment]
@zsurocking LGTM

File: android-interop-testing/app/build.gradle [hunk 1]
```diff
@@ -47,21 +47,19 @@ protobuf {
             task.plugins {
                 grpc {
                     // Options added to --grpc_out
-                    option 'nano'
+                    option 'nano=true'
```
Comment (ejona86) [inline android-interop-testing/app/build.gradle]: Revert this change. It shouldn't be necessary and it was purposefully changed to `'nano'`

**Comment by ejona86** [comment]
@zsurocking, LGTM.

I think the android example is broken as well.

File: android-interop-testing/app/build.gradle (hunk 1 above)
Comment (zsurocking) [inline android-interop-testing/app/build.gradle]: Oops. Good catch, thanks.
